phenol-annotations/src/test/java/org/monarchinitiative/phenol/annotations/formats/hpo/category/HpoCategoryMapTest.java

```diff
@@ -136,6 +137,24 @@ public void testTwoCategories() {
     assertTrue(catlist.stream().anyMatch(cat -> cat.id().equals(INHERITANCE_ID)));
   }
 
+  /**
+   * Add terms from INHERITANCE and from LIMBS_ID and CATEGORY match
+   */
+  @Test
+  public void testCategoryExact(){
```

Comment:
This test seems to be new. Does it accomplish anything? Why did you add a test that will eventually be removed? A mis-commit?